sed/core/config.py

```diff
@@ -1,60 +1,39 @@
-"""This module contains a settings/config library for loading yaml/json files into
-dicts
-
+"""This module contains a config library for loading yaml/json files into dicts
 """
 import json
-import os
-from importlib.util import find_spec
 from pathlib import Path
 from typing import Union
 
 import yaml
 
-package_dir = os.path.dirname(find_spec("sed").origin)
+DEFAULT_CONFIG_PATH = Path(__file__).resolve().parent / "default.yaml"
```

Comment:
I find it more systematic to base everything onto the base folder of the package (package_dir), rather than the individual files of the package...)

Reply:
I reverted to the old systematics, and kept the config folder containing the default config. It might also contain example config files for the different experiments in the future.